app/controllers/api/storage.php

```diff
@@ -171,18 +172,18 @@
             throw new Exception('Bucket not found', 404);
         }
 
-        $read ??= $bucket->getAttribute('$read', []); // By default inherit read permissions
-        $write ??= $bucket->getAttribute('$write',[]); // By default inherit write permissions
+        $read??=$bucket->getAttribute('$read', []); // By default inherit read permissions
+        $write??=$bucket->getAttribute('$write', []); // By default inherit write permissions
```

Comment:
What is the `??` syntax used for in this way?

Reply:
It's one of @TorstenDittmann 's suggestion and a good one
https://stackoverflow.com/questions/59102708/what-is-null-coalescing-assignment-operator-in-php-7-4

Reply:
```suggestion
        $write ??= $bucket->getAttribute('$write', []); // By default inherit write permissions
```